app/services/search/postgres/feed.rb

```diff
@@ -0,0 +1,103 @@
+module Search
+  module Postgres
+    class Feed
+      FIELDS_TO_EXCLUDE_FROM_SERIALIZATION = %i[
+        approved body_text experience_level_rating
+        experience_level_rating_distribution featured featured_number
+        hotness_score language published reactions_count score
+      ].freeze
+
+      def self.search_documents(
+        term: nil, approved: nil, class_name: nil, id: nil, sort_by: nil, sort_direction: nil, published_at: nil,
+        page: 1, per_page: 60
+      )
+        page = (page || 1).to_i + 1 # NOTE: the frontend starts at 0, need to fix this
+        per_page = [(per_page || 60).to_i, 100].min
+
+        relation = if term.present?
+                     relation = Search::Multisearch.call(term)
+                       .with_pg_search_highlight
+                       .with_pg_search_rank
+                       .includes(
+                         searchable: [
+                           {  podcast_episode: {  podcast: :creator } },
+                           { articles: :organization },
+                           :tag_taggings, :tags, :comments
+                         ],
```

Comment:
the local log is a forest of bullet complaining about eager loading and stuff. Hopefully this will disappear when we copy data in `pg_search_documents` during indexing over having to use the serializers to fetch 3 indirections of associations